frontend-dashboard/src/hooks/useSocket.jsx

```diff
@@ -1,15 +1,27 @@
 import { useEffect, useRef } from "react";
+import { useMovementStore } from "@/stores/useMovementStore";
 
 export default function useSocket() {
   const socketRef = useRef(null);
+  const setPitch = useMovementStore((state) => state.setPitch);
+  const setYaw = useMovementStore((state) => state.setYaw);
```

Comment:
```suggestion
  const { setPitch, setYaw } = useMovementStore();
```
동일한 스토어 파일에서 데이터를 불러오기 때문에, 
구조 분해 할당을 활용하면 코드 중복을 줄일 수 있을 것 같습니다!

Reply:
네 감사합니다!
위와 같이 수정하겠습니다~!